pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -1664,6 +1667,8 @@
     },
     "System.Drawing.Common": {
       "InboxOn": {
+        "netcoreapp2.0": "4.0.0.0",
```

Comment:
Ditto

Reply:
OK. And what about the version numbers? Will it be 4.0.0.0?

Reply:
Hm. Probably 4.0.1.0 I think.

Reply:
> Hm. Probably 4.0.1.0 I think.
Yup: https://github.com/dotnet/corefx/blob/master/src/System.Drawing.Common/Directory.Build.props#L5

Btw, for now it will be "netcoreapp2.1": "4.0.1.0" because we haven't re-mapped our netcoreapp alias to 3.0: https://github.com/dotnet/corefx/blob/master/tools-local/targetgroups.props#L157 and the build would fail if we don't specify it will be inbox in netcoreapp2.1